components/NewPersonView/Layout.tsx

```diff
@@ -67,6 +68,36 @@ const Layout = ({ person, children }: Props): React.ReactElement => {
   const { isFeatureActive } = useFeatureFlags();
   const [addFormOpen, setAddFormOpen] = useState<boolean>(false);
 
+  const { getConfigValue } = useAppConfig();
+
+  let workflowsNavLink;
+  let startWorkflowNavLink;
+
+  try {
+    workflowsNavLink = (
+      <NavLink
+        href={`${
+          getConfigValue('workflowsPilotUrl') as string
+        }?social_care_id=${person.id}`}
+      >
+        Workflows
+      </NavLink>
+    );
+    startWorkflowNavLink = (
+      <a
+        href={`${getConfigValue(
+          'workflowsPilotUrl'
+        )}/workflows/new?social_care_id=${person.id}`}
+        className="lbh-link lbh-body-s lbh-link--no-visited-state"
+      >
+        Start workflow
+      </a>
+    );
+  } catch (error) {
+    workflowsNavLink = null;
+    startWorkflowNavLink = null;
+  }
```

Comment:
Same comment about the `try`/`catch`, this feels wrong. Maybe worth discussing so I can understand what you were aiming for?